.env.prod

```diff
@@ -1,7 +1,6 @@
 NODE_ENV=production
-VUE_APP_BASE_URL=https://gokb.org
 VUE_APP_API_BASE_URL=https://gokb.org/gokb
-VUE_APP_YGOR_BASE_URL=https://gokb.org/ygor
+VUE_APP_PUBLIC_BASE_URL=https://gokb.org/gokb-ui
```

Comment:
The `VUE_APP_BASE_URL` is not used anywhere else anymore, right?
Keeping it in dependency to a server setup is not needed anymore.

Keeping variables in two places is less transparent and error-prone.
Managing variables outside of this repository necessarily needs full, transparent and prompt communication inside the team about what is to be done and how. Otherwise it is just building pitfalls. 
I can't see any major advantage of shifting the configurations from inside of this repo to an external place, as it is even not less work in the end.

This comment also applies to the other cases below.

Reply:
The existence of the variable is checked for deciding if the copy icon should be presented at all, so we don't generate invalid links that are not capable of redirecting via content negotiation. Maybe it should be something more descriptive like `VUE_APP_STABLE_URL_BASE` though. If such a feature is not available, the user may as well copy the current browser address anyways. I will add this to a variable description in the readme.